src/wui/minimap.cc

```diff
@@ -329,7 +359,7 @@ bool MiniMap::handle_mousewheel(int32_t x, int32_t y, uint16_t modstate) {
 	return true;
 }
 
-constexpr uint16_t kCurrentPacketVersion = 1;
+constexpr uint16_t kCurrentPacketVersion = 2;
```

Comment:
Do we need to increase this? It looks like the ships layer was added without bumping the packet version and it didn't cause any problem. But if it's increased, please add compatibility code.

Reply:
not sure I'll decide ig this is ready at the end

Reply:
There is no need to increase the version since older clients are also able to load the new-style format. Bits that are set here but not present in older versions will just be silently ignored.

Reply:
ok I came to the same conclusion